docs/02-app/02-api-reference/08-next-cli.mdx

```diff
@@ -91,26 +111,138 @@ You can also set the hostname to be different from the default of `0.0.0.0`, thi
 npx next dev -H 192.168.1.2
 ```
 
+### Turbo (beta)
+
+[Turbopack](https://turbo.build/pack) (beta), our new bundler we're testing and stabilizing through Next.js, helps speed up local iterations while working on your Next.js application.
+
+To use Turbopack in development mode, add the `--turbo` option like so:
+
+```bash filename="Terminal"
+npx next dev --turbo
+```
+
 ### HTTPS for Local Development
 
 For certain use cases like webhooks or authentication, it may be required to use HTTPS to have a secure environment on `localhost`. Next.js can generate a self-signed certificate with `next dev` as follows:
 
 ```bash filename="Terminal"
-next dev --experimental-https
+npx next dev --experimental-https
 ```
 
 You can also provide a custom certificate and key with `--experimental-https-key` and `--experimental-https-cert`. Optionally, you can provide a custom CA certificate with `--experimental-https-ca` as well.
 
 ```bash filename="Terminal"
-next dev --experimental-https --experimental-https-key ./certificates/localhost-key.pem --experimental-https-cert ./certificates/localhost.pem
+npx next dev --experimental-https --experimental-https-key ./certificates/localhost-key.pem --experimental-https-cert ./certificates/localhost.pem
 ```
 
 `next dev --experimental-https` is only intended for development and creates a locally-trusted certificate with `mkcert`. In production, use properly issued certificates from trusted authorities. When deploying to Vercel, HTTPS is [automatically configured](https://vercel.com/docs/security/encryption) for your Next.js application.
 
+## Build
+
+`next build` creates an optimized production build of your application. The output displays information about each route.
+
+- **Size** – The number of assets downloaded when navigating to the page client-side. The size for each route only includes its dependencies.
+- **First Load JS** – The number of assets downloaded when visiting the page from the server. The amount of JS shared by all is shown as a separate metric.
```

Comment:
Before jumping into **size** and **First Load JS**, maybe there could be a small snippet showing what the output looks like?

Reply:
<picture data-single-emoji=":done:" title=":done:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/done/556da3823bdfab37.png" alt=":done:" align="absmiddle"></picture> !